desc/configuration.py

```diff
@@ -364,6 +365,12 @@ def __init__(
         elif current is not None:
             raise TypeError("Got unknown current profile {}".format(current))
 
+        # remove unused profile from I/O attributes
+        if self.iota is None:
+            self._io_attrs_.remove("_iota")
```

Comment:
Is this the list.remove() error we're seeing in the pytests 3.9?

Reply:
commenting out lines 380-83 fixes the errors @ddudt

Reply:
Can't comment at it since the file was not changed. But in `tests.test_configuration.TestConstructor testMethod=test_defaults` the line `self.assertIsInstance(eq.iota, PowerSeriesProfile)` is tripping a test since eq.iota is `None` and not type `PowerSeriesProfile`.

Reply:
Those lines are removed for now, but this is still a problem. The perturbation function expects an Equilibrium to have all of the attributes listed in `Equilibrium._io_attrs_`, and this is causing one of the tests to fail.